.travis/create_artifacts.sh

```diff
@@ -31,6 +31,13 @@ echo "--- Initialize git configuration ---"
 git checkout "${1-master}"
 git pull
 
+if [ "$RELEASE_CHANNEL" = stable ]; then
```

Comment:
Please follow the conventions of the rest of the script

```suggestion
if [ "${RELEASE_CHANNEL}" == stable ]; then
```